addons/website_livechat/models/mail_channel.py

```diff
@@ -50,7 +50,7 @@ def _get_visitor_history(self, visitor):
         :param visitor: website.visitor of the channel
         :return: arrow separated string containing navigation history information
         """
-        recent_history = visitor.visitor_page_ids[:3] if len(visitor.visitor_page_ids) >= 3 else visitor.visitor_page_ids
+        recent_history = visitor.website_track_ids[:3] if len(visitor.website_track_ids) >= 3 else visitor.website_track_ids
```

Comment:
`recent_history = visitor.website_track_ids[:3] if len(visitor.website_track_ids) >= 3 else visitor.website_track_ids
`
==>

`recent_history = visitor.website_track_ids[:3]`

Reply:
I don't know who wrote this but he seemed drunk, tired, or both !
Oh wait.. 
Good one!